setup.py

```diff
@@ -32,6 +32,13 @@ def _get_long_description():
     author_email="engineering@asidatascience.com",
     url="https://github.com/ASIDataScience/dash-bootstrap-components",
     packages=find_packages(),
-    install_requires=["dash"],
+    install_requires=["dash", "pandas"],
```

Comment:
I actually don't think we should rely on pandas at installation time for a helper function for a single component. If a user wants to use the `generate_table_from_df`, they already have pandas installed.

While there are fairly complex ways to specify optional dependencies, an easy route might just be to add a lin in the docs about how this requires pandas to be installed.

Reply:
That's a fair point, I'll remove it.